Makefile

```diff
@@ -0,0 +1,16 @@
+all: bandit black flake8 isort pylama
+
+bandit:
+	pipenv run bandit -r asaps tests --skip B101
```

Comment:
Let's not run bandit on tests since we know those are likely to have security errors and that code won't get run in production so it's ok. You can also remove the skip, as that's something we do want to check for in production code.